src/server.ts

```diff
@@ -31,6 +31,7 @@ import {
   WorkspaceContextService,
   SchemaConfiguration,
   SchemaPriority,
+  LanguageService,
```

Comment:
`LanguageService` is unused, can you delete it?

Reply:
Once this is removed the PR LGTM